File: Python/1573_FabricaDeChocolate.py
import math
while True:
    a, b, c = map(int, input().split())

    if a == 0 and b == 0 and c == 0:
        break
    else:
        volume = (a * b * c)

        cubo = math.floor(volume ** (1/3))

        print(int(cubo))

File: Java/_1573_FabricaDeChocolate.java
import java.util.Scanner;

public class _1573_FabricaDeChocolate {
    public static void main(String[] args) {
        
        try (Scanner sc = new Scanner(System.in)) {
            
            while (true) { 
                int a = sc.nextInt();
                int b = sc.nextInt();
                int c = sc.nextInt();

                if (a == 0 && b == 0 && c == 0) {
                    break;
                }

                int volume = a * b * c;

                double cubo = Math.cbrt(volume);
                
                int resultado = (int) Math.floor(cubo);
                
                System.out.println(resultado);
            }
        }
    }
}